merlin/models/tf/inputs/embedding.py

```diff
@@ -150,6 +150,182 @@ def from_config(cls, config):
 
         return cls(dim, *schema, **config)
 
+@tf.keras.utils.register_keras_serializable(package="merlin.models")
+class SOKEmbedding(EmbeddingTableBase):
+    """
+    dim: int The last dimension of the variable
+    vocab_sizes: list, rows of the variable list 
+    initializer: string, a list of dict {"indices":numpy.array, "value": numpy.array} = "uniform"
+            When it's string, it specifies the initializer used to generate initial values.
+            When it's list of numpy.array, its shape must be [vocab_size[i], embedding_vec_size],
+            and will be used as the initial indices and value.
+    use_dynamic_variable: bool = "False" use sok.DynamicVariable or sok.Variable
```

Comment:
Would be great to elaborate a bit more about what the difference is between the 2 types of variables.